### ISSUE TITLE
Random Packet handling error's

### ISSUE THREAD

**Issue by Raphael065** [WaterfallMC/Waterfall#853]

**Comment by Janmm14** [comment]
If you have ViaVersion on bungee/waterfall, please try to put it on the backend spigot-based servers instead.

**Comment by Raphael396** [comment]
I have ViaVersion only on the sub-servers and not on the Bungeecord server

**Comment by SpigotRCE** [comment]
Ideally all of your via plugins should be kept on either the proxy or the server, if you put it redundantly it will raise errors and will be inconsistent.

**Comment by SpigotRCE** [comment]
And after all, move to velocity now as waterfall has reached its eol

**Comment by jquery-package** [comment]
I'm getting the same error and found a way to replicate it. It's related to SpigotMC/BungeeCord#3666, but it looks differently on Waterfall.

You can catch this by quickly changing servers multiple times, for example by using plugin like this - https://pastebin.com/cP7CT6r1. You'll also need to use MC client > 1.20.2.

**Comment by SpigotRCE** [comment]
I don't think so, the BungeeCord issue was related with server sending a packet which was not supposed to be sent during configuration, the issue which you are getting is the packet being larger than expected, do you have any Via plugins?
